package.json

```diff
@@ -106,8 +108,8 @@
   },
   "scripts": {
     "check-licenses": "wp-scripts check-licenses --prod --gpl2",
-    "build": "wp-scripts build",
-    "dev": "wp-scripts start",
+    "build": "wp-scripts build --webpack-no-externals",
+    "dev": "wp-scripts start --webpack-no-externals",
```

Comment:
To ensure this uses the npm package version in `package.json`, not whatever is in `wp.blocks`, `wp.components`, etc...